Z-Wave Regression: Fix state attributes of lights and switches.

lukas-hetzenecker opened the pull request:
Pull request #982 unfortunately broke the Zwave Dimmer component.

Light and ZWaveDeviceEntity both have overwritten the property state_attributes
Make sure to include the values of both superclasses - the light component will look in
device_state_attributes for device specific attributes. These are now the state attributes of the zwave device

lukas-hetzenecker commented:
CI Build: I see no reason in the python documentation ( https://docs.python.org/3/library/functions.html?highlight=property#property ) why the fget method of a property should have been removed.

lukas-hetzenecker commented:
The SwitchDevice seems to be affected too, will update the PR later today

balloob commented:
Auch, that's nasty. It's actually breaking parts of the switch too.

I would propose we solve it the other way around.

Rename `state_attributes` on `ZWaveDeviceEntity` to `device_state_attributes` and add this method to ZWaveSensor:
```python
@property
def state_attributes(self):
    return self.device_state_attributes
```

lukas-hetzenecker committed:
Regression: The device specific attributes of ZWave devices got lost.
Light and ZWaveDeviceEntity both have overwritten the property state_attributes
This includes the device specific attributes in the state_attributes again

lukas-hetzenecker commented:
@balloob - PR is updated, includes fix for SwitchDevice

balloob commented:
Perfect! :dolphin:
